app/models/project.rb

```diff
@@ -51,6 +51,15 @@ class Project < ActiveRecord::Base
     },
     ignoring: :accents
 
+  pg_search_scope :search_trm,
+    against: "name",
+    using: :trigram,
+    ignoring: :accents
+
+  def self.pg_search term
```

Comment:
Let's create a spec for this. We can mock the return of search_tsearch(term) and have a context for the empty array and a context for an array containing one item.